lua/lsp/manager.lua

```diff
@@ -0,0 +1,82 @@
+local M = {}
+
+function M.get_all_supported_servers()
```

Comment:
Yeah this list isn't even used currently, I had it generated from lspconfig. The functions you're looking at is a bit deceiving as it will actually only give you servers _you have already configured_.